### PR TITLE
fix: 11847 configuring UTF-8 encoding as default to SMTP plugin

### PR DESCRIPTION
**Pull request by appsmithguru**
Description

SMTP encoding issue.
https://github.com/appsmithorg/appsmith/issues/11847

The default encoding by MimeMessage is ascii and now its been configured to UTF-8 to support some characters

Checklist:

- [ ]  My code follows the style guidelines of this project
- [ ]  I have performed a self-review of my own code
- [ ]  My changes generate no new warnings
- [ ]  New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/server/appsmith-plugins/smtpPlugin/src/test/java/com/external/plugins/SmtpPluginTest.java [hunk 1]
```diff
@@ -9,16 +9,21 @@
 import com.appsmith.external.models.DatasourceConfiguration;
 import com.appsmith.external.models.DatasourceTestResult;
 import com.appsmith.external.models.Endpoint;
+import jakarta.mail.*;
```
Comment (sumitsum) [inline app/server/appsmith-plugins/smtpPlugin/src/test/java/com/external/plugins/SmtpPluginTest.java:12]: Nit: Can you pleaser replace the wild import with specific imports ? i.e. avoid using `*`.

**Review by sumitsum — DISMISSED** [review]

**Comment by appsmithguru** [comment]
This code changes are doesn't demand cypress test, so No test added

**Pull request merged**